features/topic_id.feature

```diff
@@ -0,0 +1,81 @@
+Feature: Behaviour of application when doing the GET /topics/{id} endpoint, using a stripped down version of the database
+
+    Scenario: [Test #3] GET /topics/economy in public mode
+        Given I have these topics:
+            """
+            [
+                {
+                    "id": "economy",
+                    "current": {
+                        "id": "economy",
+                        "state": "published"
+                    },
+                    "next": {
+                        "id": "economy",
+                        "state": "published"
+                    }
+                }
+            ]
+            """
+        When I GET "/topics/economy"
+        Then the HTTP status code should be "200"
+        And the response header "Content-Type" should be "application/json; charset=utf-8"
+        And I should receive the following JSON response:
+            """
+            {
+                "state": "published",
+                "id": "economy"
+            }
+            """
+
+    Scenario: [Test #4] Receive not found when doing a GET for a non existant topic in public mode
+        Given I have these topics:
```

Comment:
Ive just realised for this test we dont even need an **I have these topics:** step as we asserting against a non-existant topic

Reply:
Done